client/dashboard/profile-wizard/steps/industry.js

```diff
@@ -67,16 +69,20 @@ class Industry extends Component {
 	onChange( slug ) {
```

Comment:
Perhaps renaming this `onIndustryChange` or `onCheckboxChange` makes sense now that there are multiple input handlers.

Reply:
I will change it too.